content/doc/book/blueocean/dashboard.adoc

```diff
@@ -19,105 +19,78 @@ endif::[]
-interface.
+The Dashboard includes the blue link:../getting-started#navigation-bar[navigation bar] along the top of the interface.
 
-This bar is divided into two sections - a common section along the top and a
-contextual section below. The contextual section changes depending on the
-current Blue Ocean page you are viewing.
+This bar is divided into two sections:
 
-When viewing the Dashboard, the navigation bar's contextual section includes
-the:
+* A common section along the top 
+* A contextual section below
+** The contextual section changes depending on the Blue Ocean page you are viewing.
 
-* *Search pipelines* field to filter the link:#pipelines-list[Pipelines list]
-  to show items containing the text you enter into this field.
-* *New Pipeline* button, which begins the
-  link:../creating-pipelines[create a Pipeline] process.
+When viewing the Dashboard, the navigation bar's contextual section includes:
 
+* *Search pipelines*: This field allows users to filter the link:#pipelines-list[Pipelines list] to match the text you enter into this field.
+* *New Pipeline*: This option begins the process of link:../creating-pipelines[creating a Pipeline].
 
 == Pipelines list
 
-The "Pipelines" list is the Dashboard's default list, and upon accessing Blue
-Ocean for the first time, this is the only list shown on the Dashboard.
+The *Pipelines* list is the Dashboard's default list, and upon accessing Blue Ocean for the first time, the only list shown on the Dashboard.
 
-This list shows the overall state of each Pipeline configured on the Jenkins
-instance (which can also include other Jenkins items). For a given item in this
-list, the following information is indicated:
+This list shows the overall state of each Pipeline configured on the Jenkins instance.
+This list can include other items configured on the Jenkins instance.
+For a given item in this list, the following information is displayed:
 
 * The item's *NAME*
 * The item's link:#pipeline-health[*HEALTH*]
-* The numbers of **BRANCH**es and pull requests (**PR**s) of the
-  Pipeline's source control repository which are passing or failing
-* A star, indicating whether or not the default/main branch of the item has been
-  manually added to your current Jenkins user's <<favorites-list>>.
+* The numbers of *BRANCHES* and pull requests *(PRs)* of the Pipeline's source control repository that are passing or failing.
+* A &#9734;, indicating whether or not the item has been manually added to your current Jenkins <<favorites-list>>.
 
-Clicking on an item's star will toggle between:
+Selecting an item's &#9734; will toggle between:
 
-* Adding the default branch of the item's repository to your current user's
-  Favorites list (indicated by a solid "&#9733;"), and
-* Removing the item's default branch from this list (indicated by an outlined
-  "&#9734;").
+* Adding the default branch of the item's repository to your Favorites list, which is indicated by a solid &#9733;.
```

Comment:
Is this term in the UI? (Sorry I can't see it.)
```suggestion
* Adding the default branch of the item's repository to your *Favorites* list, which is indicated by a solid &#9733;.
```

Reply:
oh right! I removed the bold since it's not actually labeled in the UI